manager/src/main/java/com/redhat/service/bridge/manager/actions/sendtobridge/SendToBridgeActionTransformer.java

```diff
@@ -33,4 +47,9 @@ public BaseAction transform(BaseAction action, String bridgeId, String customerI
 
         return transformedAction;
     }
+
+    static String getBridgeWebhookUrl(String bridgeEndpoint) throws MalformedURLException {
```

Comment:
Can we please move this logic to the `Bridge` class for now? So something like:

```
getIngressUrl() // returns endpoint + /events
getPlainIngressUrl() // returns endpoint + /events/plain
```

My concern is that we're hiding  information that should be available via our API. By having to do this here, we're basically saying that our API doesn't currently expose enough information to clients for them to be able to interact with OpenBridge without further information e.g. they need to know that `/events` is the correct suffix to use.

I'd like us to consider updating the API to expose both endpoints so that our API is fully navigable. That's beyond the scope of this PR, but I think centralising the logic to get the endpoint onto the `Bridge` is a good first step.
I'll create an issue in our backlog to

Reply:
What about creating a dedicated task for these changes? This PR is meant to be a fix for [the initial MGDOBR-134 PR](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/141) and I'd like to fix it according to the current underlying implementation (in which the API returns the base endpoint for the bridge).

Reply:
That's fair @kostola . FYI, issue raised as https://issues.redhat.com/browse/MGDOBR-178